devices.js

```diff
@@ -5314,6 +5314,25 @@ const devices = [
             await configureReporting.batteryVoltage(endpoint);
         },
     },
+    {
+        zigbeeModel: ['motionv5'],
+        model: 'STS-IRM-251',
+        vendor: 'SmartThings',
+        description: 'Motion sensor (2017 model)',
+        supports: 'occupancy and temperature',
+        fromZigbee: [
+            fz.temperature, fz.ias_occupancy_alarm_2,
+            fz.ias_occupancy_alarm_1, fz.battery_3V,
```

Comment:
I've missed this, but either `ias_occupancy_alarm_2` or `ias_occupancy_alarm_1` should be removed. Please check which. Probably `ias_occupancy_alarm_2`.

Reply:
ias_occupancy_alarm_1 was the correct one, ias_occupancy_alarm_2 does not work.
Creating a new pull request.

Reply:
Probably the same for motionv4, but I don't have it, so I can't confirm.